src/main/java/net/wurstclient/altmanager/LoginManager.java

```diff
@@ -25,7 +26,18 @@ public static String login(String email, String password)
 		YggdrasilUserAuthentication auth =
 			(YggdrasilUserAuthentication)new YggdrasilAuthenticationService(
 				Proxy.NO_PROXY, "").createUserAuthentication(Agent.MINECRAFT);
-		
+
+		/*
+		  Automatically changing the status from mojang/thealtening, depending on the account format
+		  It won't update the service, if it's the same though.
+		 */
+		if(email.contains("@alt.com") && email.contains("-")) {

```

Comment:
I've just checked the domain alt.com lol ... That was unexpected

Reply:
Oh, thealtening tokens use @alt.com as their "email" suffix, it's nothing else. The domain is some porn shit (lmao!).